tests/tests_resize.yml

```diff
@@ -428,3 +450,118 @@
+                  type: lvm
+                  disks: "{{ unused_disks }}"
+                  volumes:
+                    - name: test1
+                      fs_type: xfs
+                      size: "{{ volume_size_after }}"
+                      mount_point: "{{ mount_location }}"
+
+          - name: Verify role results
+            include_tasks: verify-role-results.yml
+
+          - name: Repeat for idempotency test
+            include_role:
+              name: linux-system-roles.storage
+            vars:
+              storage_pools:
+                - name: foo
+                  type: lvm
+                  disks: "{{ unused_disks }}"
+                  volumes:
+                    - name: test1
+                      fs_type: xfs
+                      size: "{{ volume_size_after }}"
+                      mount_point: "{{ mount_location }}"
+
+          - name: Verify role results
+            include_tasks: verify-role-results.yml
+
+          - name: Test for correct handling of shrinking (not supported by XFS)
+            include_tasks: verify-role-failed.yml
+            vars:
+              __storage_failed_regex: volume.+cannot be resized.+
+              __storage_failed_msg: >-
+                Unexpected behavior w/ invalid volume size
+              __storage_failed_params:
+                storage_pools:
+                  - name: foo
+                    disks: "{{ unused_disks }}"
+                    volumes:
+                      - name: test1
+                        fs_type: xfs
+                        size: "{{ volume_size_before }}"
+                        mount_point: "{{ mount_location }}"
+
+          - name: Test for correct handling of acceptable size difference (slightly smaller than min)
+            block:
+              - name: >-
+                  Try to resize LVM volume size to disk size - 1.5 % (less than 2 % than
+                  minimum size should be tolerated)
+                include_role:
+                  name: linux-system-roles.storage
+                vars:
+                  storage_pools:
+                    - name: foo
+                      disks: "{{ unused_disks }}"
+                      volumes:
+                        - name: test1
+                          fs_type: xfs
+                          size: "{{ acc_small_size }}"
+                          mount_point: "{{ mount_location }}"
+
+          - name: Verify role results
+            include_tasks: verify-role-results.yml
+
+        always:
+          - name: Clean up
+            include_role:
+              name: linux-system-roles.storage
+            vars:
+              storage_pools:
+                - name: foo
+                  disks: "{{ unused_disks }}"
+                  state: absent
+                  volumes:
+                    - name: test1
+                      size: "{{ volume_size_before }}"
+                      mount_point: "{{ mount_location }}"
+
+      - name: Verify role results
```

Comment:
If this verify is meant to be done in the `always` block it should be indented at the same level as the `Clean up` task.